main.less

```diff
@@ -0,0 +1,3 @@
+/*!
```

Comment:
same question about .less files, is this still going to be the pattern?  Is this needed if it's empty, and if it can go, I guess it gets taken out of the package file.